Library/Homebrew/cmd/install.rb

```diff
@@ -206,6 +207,10 @@ def install
         ofail e.message
         query = query_regexp(e.name)
 
+        ohai "Searching for migrated formulae..."
+        migrations = search_for_migrated_formula(e.name)
```

Comment:
How long does search_for_migrated_formula take to run? I wonder if we can skip this message.